### PR TITLE
[Doc] 4.3.0 Release Notes

### PR DESCRIPTION
**Pull request by Fuzzbawls**
Finalized release notes for 4.3.0

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -32,61 +36,190 @@ PIVX Core should also work on most other Unix-like systems but is not frequently
...
 
 ### RPC/REST
-
-### Wallet
+ - #1753 `e288a4508b` [Trivial] [RPC] Fix listcoldutxos help text (JSKitty)
+ - #1828 `4fc36b59ee` [RPC][BUG] Fix ActivateBestChain calls in reconsider(invalidate)block (random-zebra)
+ - #1831 `28509bf9e8` [RPC] re introducing filtering args in getbalance (furszy)
+
+### Build Systems
+ - #1553 `fddf765132` [Build] Sapling Foundations (Build System + ZIP32) (Fuzzbawls)
```
Comment (furszy) [inline doc/release-notes.md:153]: (furszy)

Comment (Fuzzbawls) [reply doc/release-notes.md]: fixed...the script that generates this list doesn't handle multi-author PRs very well :P

**Review by furszy — APPROVED** [review]
ACK dfd3f06

**Review by random-zebra — APPROVED** [review]
ACK dfd3f0683f0c6384201a89770bfe7ae70da644af

**Pull request merged**